src/job/MomoJob.ts

```diff
@@ -2,9 +2,32 @@ export type Handler = () => Promise<string | undefined | void> | string | undefi
 
 export interface MomoJob {
   handler: Handler;
+  schedule: IntervalSchedule | CronSchedule;
   name: string;
-  interval: string;
-  firstRunAfter?: number;
   concurrency?: number;
   maxRunning?: number;
 }
+
+export interface IntervalSchedule {
+  interval: string;
+  firstRunAfter: number;
```

Comment:
can we make this optional again and default to 0, as it was before?

Reply:
the readme now tells you to use the builder and the builder allows to omit it and defaults to 0,

but I guess it would be nicer if it would also be optional if you decide to not use the builder ...

Reply:
Sure!